Gui adding

milbrain opened the pull request:
Added GUI, merged with Server-functionality

Commit:
Add first GUI

Commit:
Add Layout changees

Commit:
Try to add table for 1. row: Clients, 2. row LEDs (filled rects)

nicolas93 committed:
Get List of Clients

nicolas93 committed:
Merge branch 'gui_adding' of https://github.com/studnitz/octopus into gui_adding

Commit:
Add red LED support

Commit:
Merge branch 'gui_adding' of https://github.com/studnitz/octopus into gui_adding

Commit:
Fix red LED-List-Bug, add Tab-Widget to GUI

Commit:
Finish US #29 (Add Mulitcolored-LED display functionality to GUI)

Commit:
Merge GUI-Server with TCP-Server

nicolas93 committed:
gitignore

nicolas93 committed:
gitignore

nicolas93 committed:
m

Commit:
Merge branch 'master' of github.com:studnitz/octopus into gui_adding

nicolas93 committed:
Get List of Clients in main.cpp

nicolas93 committed:
Merge branch 'gui_adding' of https://github.com/studnitz/octopus into gui_adding

Commit:
Update qt Server in new dir structure

studnitz commented:
Please delete the *pro.user-file in the qt server with gui directory. Also we should consider having the GUI-Controller and View in a directory called gui. 

Copy one of the innermost .gitignores to the main .gitignore.

nicolas93 committed:
Change Client-List from dummydata to real data

nicolas93 committed:
Merge branch 'gui_adding' of github.com:studnitz/octopus into gui_adding

nicolas93 committed:
Fix Rowcount in Clientlist

Commit:
Update .gitignore

Commit:
Update Working TestCase for communication

Commit:
Add Videoplayer (not implemented funcionality yet)

Commit:
Merge branch 'gui_adding' of github.com:studnitz/octopus into gui_adding

Commit:
Do necessary merge

Commit:
Merge branch 'gui_adding' of https://github.com/studnitz/octopus into gui_adding

Commit:
Restructure directory structure

The structure of the directories was messed up, now we have a more
modular approach with several *.pro files for better organization.

Commit:
Remove qt directory

Commit:
Remove extra .gitignores

Commit:
Add Pause / Stop / Playback-Functionality for arbitrary files
